CHANGELOG.rst

```diff
@@ -7,29 +7,35 @@ Change log
 BREAKING CHANGES
 ~~~~~~~~~~~~~~~~
 
-* **migrate to Python3.7**
-* **disable entry points and switch to ``python -m`` calls**
-* **rename ``_universum`` to ``universum``** *(configs need to be fixed)*
+* **migrate to Python3.7** due to Python2 is no longer supported
+    Python3.7 is now :doc:`required <prerequisites>` for Universum to work
+* **disable entry points** and switch to ``python3.7 -m universum`` calls
+    to avoid possible collisions, especially when using :doc:`analyzers <code_report>`
```

Comment:
Not clear.
What collisions?
Also, users probably don't know what is "entry point" in this context.
Please add migration notice.